src/PointwiseFunctions/AnalyticSolutions/GeneralRelativity/GaugePlaneWave.cpp

```diff
@@ -23,10 +23,10 @@ GaugePlaneWave<Dim>::GaugePlaneWave(CkMigrateMessage* /*msg*/) {}
 
 template <size_t Dim>
 GaugePlaneWave<Dim>::GaugePlaneWave(
-    std::array<double, Dim> wave_vector,
-    std::unique_ptr<MathFunction<1, Frame::Inertial>> profile)
-    : wave_vector_(std::move(wave_vector)),
-      profile_(std::move(profile)),
+    const std::array<double, Dim>& wave_vector,
+    const std::unique_ptr<MathFunction<1, Frame::Inertial>>& profile)
+    : wave_vector_(wave_vector),
+      profile_(profile->get_clone()),
```

Comment:
Here the move is better for the unique ptr. For the array it doesn't matter. It's possible to elide a copy by taking the pointer by value and then moving.

Reply:
Suggested changes have been applied. @nilsdeppe